library/core/src/mem/transmutability.rs

```diff
@@ -0,0 +1,39 @@
+/// Are values of a type transmutable into values of another type?
+///
+/// This trait is implemented on-the-fly by the compiler for types `Src` and `Self` when the bits of
+/// any value of type `Self` are safely transmutable into a value of type `Dst`, in a given `Context`,
+/// notwithstanding whatever safety checks you have asked the compiler to [`Assume`] are satisfied.
+#[unstable(feature = "transmutability", issue = "none")]
+#[cfg_attr(not(bootstrap), lang = "transmute_trait")]
+pub unsafe trait BikeshedIntrinsicFrom<
```

Comment:
Add the `rustc_on_unimplemented` attribute to this trait to make the diagnostics a bit more palatable

Reply:
Resolved by https://github.com/rust-lang/rust/pull/92268/commits/cac55faf764a90d3fb9a90dc0622238843dfb12c